samples/csharp/getting-started/DeepLearning_ImageClassification_TensorFlow/ImageClassification/assets/inputs/images/wikimedia.md

```diff
@@ -0,0 +1,20 @@
+Donovan Govan, [Toaster](https://commons.wikimedia.org/wiki/File:Toaster.jpg), [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/legalcode)  
+Consumer Reports, [Consumer Reports - Hamilton Beach Digital toaster](https://commons.wikimedia.org/wiki/File:Consumer_Reports_-_Hamilton_Beach_Digital_toaster.tiff), [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.en)  
+MichalPL, [Ok Toaster](https://commons.wikimedia.org/wiki/File:OK._Toaster.jpg) [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.en)  
+Hussein Kefel, [A Canoe At The_River Number 2 -Sierra Leone](https://commons.wikimedia.org/wiki/File:A_Canoe_At_The_River_Number_2_-_Sierra_Leone.JPG),  [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/legalcode)  
+Jarek Tuszyński, [Adirondacks - Rolling Pond Campground - canoe](https://commons.wikimedia.org/wiki/File:Adirondacks_-_Rolling_Pond_Campground_-_canoe.JPG), [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/legalcode)  
+Harasmode, [Cascada de Tamul, San Luis Potosí](https://commons.wikimedia.org/wiki/File:Cascada_de_Tamul,_San_Luis_Potos%C3%AD.jpg), [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.en)  
+Jon Sullivan, [Pepperoni pizza](https://commons.wikimedia.org/wiki/File:Pepperoni_pizza.jpg), This work has been released into the public domain by its author, Jon Sullivan ([PD Photo.org](http://pdphoto.org/))  
+Sapoproductions, [Pizza con huevo](https://commons.wikimedia.org/wiki/File:Pizza_con_huevo.jpg), [GNU Free Documentation License v1.2 or later](https://en.wikipedia.org/wiki/en:GNU_Free_Documentation_License) and [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/)  
+Agnieszka Kwiecień, [Spinach pizza](https://commons.wikimedia.org/wiki/File:Spinach_pizza.jpg), [GNU Free Documentation License v1.2 or later](https://en.wikipedia.org/wiki/en:GNU_Free_Documentation_License) and [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/legalcode)  
+Dickbauch, [Kaffekanne db](https://commons.wikimedia.org/wiki/File:Kaffekanne_db.jpg), [GNU Free Documentation License v1.2 or later](https://en.wikipedia.org/wiki/en:GNU_Free_Documentation_License) and [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/legalcode)  
+Skottniss, [Coffeepotsteel](https://commons.wikimedia.org/wiki/File:Coffeepotsteel.jpg), [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.en)  
+Kaffekjele, [Kaffekjele, Norsk Folkemuseum, NF.2007-0149AB](https://commons.wikimedia.org/wiki/File:Kaffekjele,_Norsk_Folkemuseum,_NF.2007-0149AB.jpg)  
```

Comment:
Kaffekjele is listed under Creative Commons Attribution-Share Alike 4.0 International